mcs/class/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -0,0 +1,262 @@
+      <Compile Include="System\Type.cs" />
+      <Compile Include="System.Globalization\GlobalizationMode.cs" />
+      <Compile Include="System.Globalization\CultureData.cs" />
+      <Compile Include="System.Globalization\CultureInfo.cs" />
+      <Compile Include="System.Collections.Generic\Comparer.cs" />
+      <Compile Include="System.Collections.Generic\EqualityComparer.cs" />
+      <Compile Include="System.Diagnostics\Debugger.cs" />
+      <Compile Include="System.Diagnostics\StackFrame.cs" />
+      <Compile Include="System.Diagnostics\StackTrace.cs" />
+      <Compile Include="System.Diagnostics.Tracing\EventSource.cs" />
+      <Compile Include="System.IO\Stream.cs" />
+      <Compile Include="System.IO\FileLoadException.cs" />
+      <Compile Include="System.Reflection\Assembly.cs" />
+      <Compile Include="System.Reflection\AssemblyName.cs" />
+      <Compile Include="System.Reflection\CustomAttributeExtensions.cs" />
+      <Compile Include="System.Reflection\FieldInfo.cs" />
+      <Compile Include="System.Reflection\MethodBase.cs" />
+      <Compile Include="System.Reflection.Emit\Stubs.cs" />
+      <Compile Include="System.Reflection.Metadata\AssemblyExtensions.cs" />
+      <Compile Include="System.Resources\ResourceManager.cs" />
+      <Compile Include="System.Runtime.CompilerServices\RuntimeHelpers.cs" />
+      <Compile Include="System.Runtime.CompilerServices\RuntimeFeature.cs" />
+      <Compile Include="System.Runtime.InteropServices\ComEventsHelper.cs" />
+      <Compile Include="System.Runtime.InteropServices\CriticalHandle.cs" />
+      <Compile Include="System.Runtime.InteropServices\InteropExtensions.cs" />
+      <Compile Include="System.Runtime.InteropServices\Marshal.cs" />
+      <Compile Include="System.Runtime.Loader\AssemblyLoadContext.cs" />
+      <Compile Include="System.Threading\EventWaitHandle.cs" />
+      <Compile Include="System.Threading\Interlocked.cs" />
+      <Compile Include="System.Threading\Mutex.cs" />
+      <Compile Include="System.Threading\Overlapped.cs" />
+      <Compile Include="System.Threading\PreAllocatedOverlapped.cs" />
+      <Compile Include="System.Threading\Semaphore.cs" />
+      <Compile Include="System.Threading\SynchronizationContext.cs" />
+      <Compile Include="System.Threading\Thread.cs" />
+      <Compile Include="System.Threading\ThreadPool.cs" />
+      <Compile Include="System.Threading\ThreadPoolBoundHandle.cs" />
+  </ItemGroup>
+  <ItemGroup>
+      <Compile Include="temp\*.cs" />
+  </ItemGroup>
+  <ItemGroup>
+      <Compile Include="..\..\build\common\SR.cs" />
+
+      <Compile Include="..\corlib\coreclr\Math.CoreCLR.cs" />
+      <Compile Include="..\corlib\coreclr\SorterArray.cs" />
+      <Compile Include="..\corlib\corefx\DateTime.cs" />
+      <Compile Include="..\corlib\corefx\SynchronizationContext.cs" />
+      <Compile Include="..\corlib\corefx\RuntimeImports.cs" />
+      <Compile Include="..\corlib\corert\ThreadPool.cs" />
```

Comment:
I had to add 
```
      <Compile Include="..\corlib\corert\RuntimeAugments.cs" />
```

because otherwise I was getting an error:

```
/Users/alekseyk/work/monovm-coreclr/mono/external/corert/src/System.Private.CoreLib/src/System/Reflection/CustomAttributeNamedArgument.cs(83,31): error CS0103: The name 'RuntimeAugments' does not exist in the current context [/Users/alekseyk/work/monovm-coreclr/mono/mcs/class/System.Private.CoreLib/System.Private.CoreLib.csproj]
```